tests/sources/test_jira.py

```diff
@@ -555,6 +559,15 @@ async def test_get_with_500_status():
                     await response.json()
 
 
+@pytest.mark.asyncio
+async def test_ping_to_custom_path_server():
+    async with create_jira_source(jira_url="http://127.0.0.1:8080/test/") as source:
+        with patch.object(
+            aiohttp.ClientSession, "get", side_effect=side_effect_function
+        ):
+            await source.ping()
```

Comment:
let's add an assertion here that checks that the pinged url is prefixed with `/test/`